spec/views/users/show.html.erb_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'User show page', type: :feature do
+  describe 'User show page process' do
+    before(:each) do
+      @user = User.create(name: 'Nurka', photo: 'https://google.com/nur.jpg', bio: 'Asks tasks',
+                          PostCounter: 2)
+      Post.create(author: @user, title: 'My first post', text: 'This is my first post')
+      Post.create(author: @user, title: 'My second post', text: 'This is my second post')
+      Post.create(author: @user, title: 'My third post', text: 'This is my third post')
+      @last_post = Post.create(author: @user, title: 'My fourth post', text: 'This is my fourth post')
+
+      visit user_path(@user.id)
+    end
+
+    it 'I can see the user\'s profile picture' do
+      expect(page.body).to include(@user.photo)
+    end
+
+    it 'I can see the user\'s username.' do
+      expect(page.body).to have_content(@user.name)
+    end
+
+    it 'I can see the number of posts the user has written.' do
+      expect(page.body).to have_content(@user.PostCounter.to_s)
+    end
+
+    it 'I can see the user\'s bio' do
+      expect(page.body).to have_content(@user.bio)
+    end
+
```

Comment:
- [ ] Good job adding the tests for the User show page. But you are missing the `test that asserts you can see the user's first 3 posts.` Kindly add this test.